js/ui/range_slider.js

```diff
@@ -297,6 +307,20 @@ const RangeSlider = Slider.inherit({
         this.option('value', [start, end]);
     },
 
+    _getActualStartValue: function() {
```

Comment:
I think it's better to implement getter and setter as a one function. Like it's done for [isTextCompositionInProgress](https://github.com/DevExpress/DevExtreme/blob/22_2/js/ui/drop_down_editor/ui.drop_down_list.js#L677) in ddList, for example.

It can simplify future debug because we have only one place where`_actualValue` prop changes